manifests/0000_90_etcd-operator_03_prometheusrule.yaml

```diff
@@ -130,6 +119,17 @@ spec:
         severity: warning
   - name: openshift-etcd.rules
     rules:
+    - alert: etcdGRPCRequestsSlow
+      annotations:
+        description: 'etcd cluster "{{ $labels.%s }}": 99th percentile of gRPC requests is {{ $value }}s on etcd instance {{ $labels.instance }} for {{ $labels.grpc_method }} method.'
```

Comment:
Are you sure you want `$labels.%s` here instead of `$labels.job`?

/hold

Reply:
good catch, thanks - fixing